examples/Actor/DemoActor/DemoActor.csproj

```diff
@@ -1,13 +1,24 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
-  <PropertyGroup>
-    <TargetFramework>net6</TargetFramework>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="..\..\..\src\Dapr.Actors.AspNetCore\Dapr.Actors.AspNetCore.csproj" />
-    <ProjectReference Include="..\..\..\src\Dapr.Actors\Dapr.Actors.csproj" />
-    <ProjectReference Include="..\IDemoActor\IDemoActor.csproj" />
-  </ItemGroup>
+    <PropertyGroup>
+        <TargetFramework>net6</TargetFramework>
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <IsPublishable>true</IsPublishable>
+        <EnableSdkContainerSupport>true</EnableSdkContainerSupport>
+        <ContainerRepository>demo-actor</ContainerRepository>
+        <ContainerBaseImage>mcr.microsoft.com/dotnet/runtime:6.0-jammy-chiseled-extra</ContainerBaseImage>
```

Comment:
Nit: can we get away without specifying a specific base image?  Given it's a sample, we shouldn't be particularly concerned about image size, etc.  I'd rather not have to remember to update this as the target framework changes.

Reply:
Done